src/Favourites/Favourites.css

```diff
@@ -0,0 +1,22 @@
+.Favourites {
+    display: block;
+}
+
+.Favourites-tabs {
```

Comment:
i actually didn't add sass because i didn't like to nest class names 😅 it makes css hard to debug, but our app is super small anyway hahaha

since you've already added it let's leave it in there!

Reply:
:highfive: